reference/providers/ecs.md

```diff
@@ -44,4 +47,68 @@ Scales the desired count for the Amazon ECS service down to 0 instances (tasks),
 Scales the desired count for the Amazon ECS service down to 0 instances (tasks), so no instances are running.
 
 ### Resize
-Scales the Amazon ECS service up or down to the desired number of instances (tasks).
\ No newline at end of file
+
+Scales the Amazon ECS service up or down to the desired number of instances (tasks).
+
+# Resource Naming
+
+As of Spinnaker 1.24.0 the ECS provider can be configured to use either Frigga or a tag based naming strategy which Spinnaker manages using its
+[Moniker](https://github.com/spinnaker/moniker) library. The naming strategy is configurable per-account and can be have a default across all accounts.
```

Comment:
grammar / "...can be have..." --> "...or applied as a default across all accounts" ?